docs/user/riboviz_outputs.md

```diff
@@ -0,0 +1,396 @@
+* [frame_proportions_per_ORF.pdf](#frame_proportions_per_ORFpdf)
+
+After a riboviz run, many output files are produced within the output directory.
+The output directory is specified by the parameter `dir_out` in `config.yaml`.
+
+There are a few output files that collect information for an entire run.
+There are many output files that are specific to each sample, which are organized into a separate subdirectory for each sample. 
+
+
+# Output files for an entire run
+
+
+## `TPMs_all_CDS_all_samples.tsv` 
+
+A tsv file with transcripts per million (tpm) for all genes from successfully processed samples. This file is produced by the script `collate_TPMs.R`. This script uses the `ORF_TPMs_and_counts.tsv` file from each processed sample and lists the tmps of each sample for each gene, allowing for comparison between samples. 
+
+```
+ORF	ANAPH_1	VEG_1
+Q0045	54.5	0.7
+Q0050	0.6	0.7
+Q0055	0.6	0.1
+Q0060	2.2	0.1
+Q0065	22.2	1
+Q0070	22.8	0.7
+```
+
+
+## `read_counts_per_file.tsv` 
+
+A [read counts file](#read-counts-file) (only if `count_reads: TRUE`).
+
+A tsv file produced by `count_reads.py`. This file lists the sample being processed, the stage of the riboviz process, the path to the file being processed at that stage, the number of reads present at each stage, and includes a description of the process. The number of reads is expected to decrease between stages as reads are filtered and trimmed.
+
+```
+SampleName	Program	File	NumReads	Description
+VEG_1	input	/exports/csce/eddie/biology/groups/wallace_rna/riboviz-emma/riboviz/riboviz/B-Sc_2012/input/SRR387871.fastq.gz	21155927	input
+ANAPH_1	input	/exports/csce/eddie/biology/groups/wallace_rna/riboviz-emma/riboviz/riboviz/B-Sc_2012/input/SRR387890.fastq.gz	10210064	input
+ANAPH_1	cutadapt	/exports/csce/eddie/biology/groups/wallace_rna/riboviz-emma/riboviz/riboviz/B-Sc_2012/tmp/ANAPH_1/trim.fq	9458608	Reads after removal of sequencing library adapters
+ANAPH_1	hisat2	/exports/csce/eddie/biology/groups/wallace_rna/riboviz-emma/riboviz/riboviz/B-Sc_2012/tmp/ANAPH_1/nonrRNA.fq	5999936	rRNA or other contaminating reads removed by alignment to rRNA index files
+```
+
+
+# Output files for each sample
+
+For each sample (`<SAMPLE_ID>`), intermediate files are produced in a sample-specific subdirectory (`<SAMPLE_ID>`).
+
+
+## `<SAMPLE_ID>_output_report.html` 
+
+This output report in .html format contains a provenance section for the sample, all the pdf output files produced by riboviz, and information on any plots not produced such as which files would be needed to produce plots in future runs. The HTML also includes a side bar to allow for navigation between figures. This file is produced by `AnalysisOutputs.Rmd`, which loads and creates all of the output graphs in HTML format.
+
+Example of the top of the HTML file:
+
+<img src="https://github.com/3mma-mack/Riboviz-honours/blob/main/riboviz_images/html_output.jpg" alt="html_output image" width="500"/>
```

Comment:
This is too brittle as if your repository goes the image link will break. Add the image to `docs/images/` then replace the URL with `<img src="../images/html_output.jpg" ... />`. Comment applies to all `<img>` links in this doc.